SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -277,6 +283,23 @@ void USpatialGameInstance::HandleOnPlayerSpawnFailed(const FString& Reason)
 	OnSpatialPlayerSpawnFailed.Broadcast(Reason);
 }
 
+constexpr static uint8 SimPlayerErrorExitCode = 10;
```

Comment:
10 was picked to avoid intersecting with special meaning exit codes as described here https://tldp.org/LDP/abs/html/exitcodes.html
Also would like to discuss whether we want to check for this exit code in the WorkerCoordinator to avoid confusion when people see their SimPlayer being restarted with an arbitrary exit code of 10.